sdk/core/azure-core/src/main/java/com/azure/core/implementation/logging/DefaultLoggerBase.java

```diff
@@ -0,0 +1,841 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.implementation.logging;
+
+import org.slf4j.Logger;
+import org.slf4j.Marker;
+
+public abstract class DefaultLoggerBase implements Logger {
+
+    /**
+     * Return the name of this <code>Logger</code> instance.
+     *
+     * @return name of this logger instance
+     */
+    @Override
+    public String getName() {
+        return null;
```

Comment:
ClientLogger passes the class name to DefaultLogger. Is it not that useful at DefaultLoggerBase layer, since we need to add private field and pass the class name to base class. I prefer to leave it alone.